example/lib/demo_features/nft/nft_binding.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:example/demo_features/nft/nft_controller.dart';
+import 'package:get/get.dart';
+import 'package:peppermint_sdk/peppermint_sdk.dart';
+
+class NftBinding extends Bindings {
+  @override
+  void dependencies() {
+    Get.lazyPut<NftRepo>(
+      () => NftRepo(
+        walletClient: GetConnect(),
+        baseUrl: 'https://peppermint-api.dev',
```

Comment:
no need to define baseUrl to repo.
should define baseUrl in GetConnect().
The same way as WalletConnect

Reply:
So you can put WalletClient in your example. And throw it to `walletClient`: